src/components/SongsAndStories/SongsAndStoriesPresentation.js

```diff
@@ -89,40 +91,35 @@ function SongsAndStoriesPresentation({
                       <h2 id="gallery-heading" className="sr-only">
                         {pluralDocType}
                       </h2>
-                      <ul
-                        role="list"
-                        className="grid grid-cols-1 gap-y-8 md:grid-cols-3 md:gap-x-6 lg:grid-cols-3 xl:grid-cols-4 xl:gap-x-8"
-                      >
-                        {items?.pages?.map((page, index) => (
-                          <React.Fragment key={index}>
+                      <ul className="grid grid-cols-1 gap-y-8 md:grid-cols-3 md:gap-x-6 lg:grid-cols-3 xl:grid-cols-4 xl:gap-x-8">
+                        {items?.pages?.map((page) => (
+                          <React.Fragment key={page.pageNumber}>
                             {page.results.length > 0 ? (
                               page.results.map((item) => {
-                                const hasCoverImage = item.photos?.length > 0
-                                const hideTextOverlay =
-                                  item?.presentationSettings?.some(
-                                    (setting) =>
-                                      setting.key === 'hideTextOverlay',
-                                  )
+                                const hasCoverImage =
+                                  item.coverVisual?.type === IMAGE
```

Comment:
The existing songs + stories grid view only used images and not videos for covers. @gmcauliffe Is that good for now or should I enhance this to support background videos as well?

Reply:
I think that is fine for now. Thanks :)